src/index.ts

```diff
@@ -0,0 +1,79 @@
+import express from "express";
+import fs from "fs";
+import { nextTick } from "process";
+import { checkNote } from "./api/checkNote";
+
+import { notesApi } from "./api/notesApi";
+import { Note } from "./config/note";
+import { dateFormatter } from "./utils/helper";
+import { validate } from "./utils/validation";
+
+const app = express();
+const jsonParser = express.json();
+const port = process.env.PORT || 3000;
+app.use(function (request, response, next) {
+  let data = `${dateFormatter()} ${request.method} ${request.url} ${request.get(
+    "user-agent"
+  )}`;
+  console.log(data);
+  fs.appendFile("server.log", data + "\n", function () {});
+  next();
+});
+app.get("/", (request, response) => {
+  response.send(`<h1>Hello </h1>`);
+});
+app.get(
+  "/api/greetings",
+  (request: express.Request, response: express.Response) => {
+    const userName = request.query.name;
+    if (userName === undefined) {
+      response.setHeader("Content-Type", "application/json");
+      response
+        .status(400)
+        .send(
+          JSON.stringify(
+            { name: null, message: "The name was not indicated" },
+            null,
+            3
+          )
+        );
+    } else {
+      response.send(`<h1>Hello ${userName}</h1>`);
+    }
+  }
+);
```

Comment:
`app.get(ROUTES.GREETING, asyncMiddleware(greetingHandler))`
And move all the logic inside `modules/greeting/controllers/getGreeting`